<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>976</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>976</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAN Simulator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Indicator</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton_IND_OFF">
         <property name="text">
          <string>Off</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton_IND_H">
         <property name="text">
          <string>Hazard</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_IND_L">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton_IND_R">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_VehicleSpeed">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Speed:</string>
        </property>
        <property name="buddy">
         <cstring>horizontalSlider_Speed</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_Speed">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Speed">
        <property name="text">
         <string>0 Kmph</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_EngineSpeed_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Fuel:</string>
        </property>
        <property name="buddy">
         <cstring>horizontalSlider_FuelLevel</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_FuelLevel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_FuelLevel">
        <property name="text">
         <string>0 %</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_EngineSpeed">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>RPM:</string>
        </property>
        <property name="buddy">
         <cstring>horizontalSlider_RPM</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider_RPM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_RPM">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Brake Pedal</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_BP_Applied">
         <property name="text">
          <string>Applied</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_BP_Released">
         <property name="text">
          <string>Released</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Select Vehicle</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblsv_make">
            <property name="text">
             <string>Make:</string>
            </property>
            <property name="buddy">
             <cstring>cb_sv_make</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_sv_make">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblsv_model">
            <property name="text">
             <string>Model:</string>
            </property>
            <property name="buddy">
             <cstring>cb_sv_model</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_sv_model">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblsv_year">
            <property name="text">
             <string>Year:</string>
            </property>
            <property name="buddy">
             <cstring>cb_sv_year</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_sv_year"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSelectVehicle">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Serial Port</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblsp_name">
            <property name="text">
             <string>COM:</string>
            </property>
            <property name="buddy">
             <cstring>cb_sp_coms</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_sp_coms">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSerialPortConn">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Seat Belt</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_SB_On">
         <property name="text">
          <string>On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_SB_Off">
         <property name="text">
          <string>Off</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionModify"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>cb_sv_make</tabstop>
  <tabstop>cb_sv_model</tabstop>
  <tabstop>cb_sv_year</tabstop>
  <tabstop>pushButtonSelectVehicle</tabstop>
  <tabstop>cb_sp_coms</tabstop>
  <tabstop>pushButtonSerialPortConn</tabstop>
  <tabstop>horizontalSlider_Speed</tabstop>
  <tabstop>horizontalSlider_RPM</tabstop>
  <tabstop>horizontalSlider_FuelLevel</tabstop>
  <tabstop>radioButton_BP_Applied</tabstop>
  <tabstop>radioButton_BP_Released</tabstop>
  <tabstop>radioButton_IND_OFF</tabstop>
  <tabstop>radioButton_IND_L</tabstop>
  <tabstop>radioButton_IND_H</tabstop>
  <tabstop>radioButton_IND_R</tabstop>
  <tabstop>radioButton_SB_On</tabstop>
  <tabstop>radioButton_SB_Off</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
